phylogenetic/rules/annotate_phylogeny.smk

```diff
@@ -42,11 +42,12 @@ rule ancestral:
 rule translate:
     """Translating amino acid sequences"""
     input:
-        tree = "results/tree_{serotype}_genome.nwk",
-        node_data = "results/nt-muts_{serotype}_genome.json",
-        reference = "config/reference_dengue_{serotype}.gb"
+        tree = "results/tree_{serotype}_{gene}.nwk",
+        node_data = "results/nt-muts_{serotype}_{gene}.json",
+        # The genbank references for the E gene are dynamically generated files located within the results folder.
```

Comment:
One minor simplification (take it or leave it) is to rename the genome reference to `config/reference_dengue_{serotype}_genome.gb` then you can drop the lambda function.

And even more minor, we can drop `_dengue` from the pathnames if you want. 

And (finally!), for my own interest, why are we dynamically creating the E gene references? To avoid adding ~5 more files?

Reply:
I agree on dropping `_dengue` from filenames and adding `_genome` to reference genome genbank files. Fixed in https://github.com/nextstrain/dengue/pull/18/commits/09db061fabcaf2c8713d9fe0d8f784c4a6e0a6c7. 

Note that the lambda expression is still required to reference paths like `config` vs `results/config`.

Reply:
> And (finally!), for my own interest, why are we dynamically creating the E gene references? To avoid adding ~5 more files?

In part yes, the other rationale behind dynamically creating the E gene GenBanks is to ensure that they remain synchronized with the existing 5 whole genome GenBank files.